CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 ## To be Released
 
+## v1.0.0
+
+### Breaking Changes
+
+* Resource `github_link`: `deploy_on_branch_change` has been removed
```

Comment:
Has it been replaced with something else? Or just removed with no solution to replace it? If there is a replacement solution, I would mention it here.

Reply:
Nope, there isn't. it was pointless in Terraform, creating a lot of side effect